### ISSUE TITLE
Add Nasdaq-ens-index

### ISSUE THREAD

**Issue by tydixon32** [Zimtente/ens-collections#114]
{
            "name": “Nasdaq ens index”,
            "slug": “Nasdaq-ens-index”,
            "description": “Nasdaq ens index AABA-ZYNE”,
            "twitter": "nasdaq_ensindex",
            "chat": "https://discord.gg/5J9xE2X7XH",
            "website": "",
            "logo": “Nasdaq-ens-index.png”,
            "csv": ["Nasdaq-ens-index.csv"]
        }

**Comment by tunnckoCore** [comment]
And what's the point of this? In mid to long term?

**Comment by tydixon32** [comment]
Hey I think it is a mid to long term opportunity with plenty of room for short term minters and flippers. I believe that stocks tickers will most likely have an actual use case with their corresponding ETH address in the near future, what that might be would be further speculation. 

As I shared in the discord nasdaq is already exploring the blockchain sourced from their own website. I think people would have a good time finding their favorite company or speculating on companies, I attached to the twitter a paste bin with the corresponding companies and people have already been minting their favorites.
